implementations/4/acme_container.json

```diff
@@ -0,0 +1,89 @@
+{
+  "spdxId": "urn:sbom-acme-container-1.3-4ab0bd4b-064b-4e36-94ac-3459a2425ba8",
+  "name": "acme-container",
+  "element": [
+    "urn:product-acme-container-1.3-58636866-50ba-4a50-a11b-818ff4f5bdc7",
+    "urn:acme.com-4fe40e24-20e3-11ee-be56-0242ac120002"
+  ],
+  "rootElement": [
+    "urn:product-acme-container-1.3-58636866-50ba-4a50-a11b-818ff4f5bdc7"
+  ],
+  "creationInfo": {
+    "specVersion": "3.0.0",
+    "created": "2023-08-16T21:24:44Z",
+    "createdBy": [
+      "urn:acme.com-4fe40e24-20e3-11ee-be56-0242ac120002"
+    ],
+    "profile": [
+      "core",
+      "software"
+    ],
+    "dataLicense": "CC0-1.0",
+    "createdUsing": [
+      "app_in_container.py"
+    ]
+  },
+  "type": "SpdxDocument",
+  "elements": {
+    "person": [],
+    "organization": [
+      {
+        "spdxId": "urn:acme.com-4fe40e24-20e3-11ee-be56-0242ac120002",
+        "type": "Organization"
+      }
+    ],
+    "package": [
+      {
+        "spdxId": "urn:product-acme-container-1.3-58636866-50ba-4a50-a11b-818ff4f5bdc7",
+        "name": "acme/acme-container",
+        "packageVersion": "1.3",
+        "type": "Package"
+      }
+    ]
+  },
+  "imports": [
+    {
+      "externalId": "urn:external-sbom-acme-app-5a6e83d0-e085-43d3-93dd-d53db6a6b19c",
+      "verifiedUsing": [
+        {
+          "hashAlgorithm": "sha256",
+          "hashValue": "248974de1b297b5167229e87143d12b128db851e75e3d05f6d1e7542ef4674dc",
+          "type": "Hash"
+        }
+      ],
+      "locationHint": "acme_compact.json",
```

Comment:
Currently, the location hint defined in the spec is `type: xsd:anyURI` which doesn't map this examples.

Personally, I think the spec should be changed to be a string as this use of the location hint makes sense